### ISSUE TITLE
Not working in Craft version 3.7.16

### ISSUE THREAD

**Issue by punani3000** [mmikkel/Reasons-Craft3#13]
Hi!

Could it be that the plugin doesn't work with Craft version 3.7.16? The plugin used to work but since version 3.7.16 no options appear when editing fields for sections and the conditions are also gone.

Thanks,
Dirk

**Comment by punani3000** [comment]
I'm using Reasons 2.2.5 and deactivated all other plugins. In the console I couldn't find any errors.

I noticed that it doesn't work when `Craft::$app->getUser->getIdentity()` is called in a module.  I have a custom module and to test it I used the following code in the `init` function (the constructor is the default one):

```
        parent::init();

        self::$instance = $this;


        if ( $user = Craft::$app->getUser() ) {
            $this->currentUser = $user->getIdentity();
        }

        return;
```

When I comment out `$this->currentUser = $user->getIdentity();` it works.

I'm using Chrome 94.0.4606.71 on macOS 11.3.1.
